sources/org.osbuild.skopeo

```diff
@@ -64,18 +64,22 @@ SCHEMA = """
 """
 
 
-def download(items, cache):
-    for image_id, item in items.items():
-        image = item["image"]
+class SkopeoSource(sources.SourceService):
+
+    content_type = "org.osbuild.containers"
+
+    # pylint: disable=arguments-renamed
+    def fetch_one(self, image_id, desc):
```

Comment:
As previous, I think `checksum` plus a `image_id = checksum` is nicer (and same number of lines ;P, if you count the comment).